.github/workflows/graalwasm-micronaut-excelice.yml

```diff
@@ -0,0 +1,48 @@
+name: Test GraalWasm Micronaut Excelice Demo
+
+on:
+  push:
+    paths:
+      - 'graalwasm/graalwasm-micronaut-excelice/**'
+      - '.github/workflows/graalwasm-micronaut-excelice.yml'
+  pull_request:
+    paths:
+      - 'graalwasm/graalwasm-micronaut-excelice/**'
+      - '.github/workflows/graalwasm-micronaut-excelice.yml'
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  run:
+    name: 'graalwasm-micronaut-excelice'
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: graalvm/setup-graalvm@v1
+        with:
+          java-version: '21'
+          distribution: 'graalvm'
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          cache: 'maven'
+          native-image-job-reports: 'true'
+
+      - name: Install Node.js & Wasm for GraalVM
+        run: |
+          gu install js wasm nodejs
+
+      - name: Package 'graalwasm-micronaut-excelice'
+        run: |
+          cd graalwasm/graalwasm-micronaut-excelice
+          chmod +x mvnw
```

Comment:
You need to make sure `mvnw` is executable on your system and then git should keep track of this. The command should not be needed.
```suggestion
```